specs/app_settings/03_technical_details.rst

```diff
@@ -0,0 +1,16 @@
+=====================
+Technical details
+=====================
+
+=====  ============  =========  ==========
+epic      title       author     created
+=====  ============  =========  ==========
+`82`_  App settings  tschubotz  2019-01-18
+=====  ============  =========  ==========
+
+.. _82: https://github.com/gnosis/safe/issues/82
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
```

Comment:
If we merge without the technical detail then maybe we can remove this file and its references what do you think?

Reply:
true